### ISSUE TITLE
New Gradle Project: Options page does not reflect workspace settings.

### ISSUE THREAD

**Issue by pfurbacher** [eclipse/buildship#696]
Buildship version: 2.2.1.v2018125-1441

I changed the workspace setting "Automatic Project Synchronization" to true to see how it affects Gradle projects.

I then created a new Gradle project using the Gradle Project wizard. I would have expected the "Options" page to show "Automatic Project Synchronization" as checked. It does not, as shown in the  attached image.

The same problem occurs with any changed workspace setting; they simply are not reflected in the project's options in the wizard.

![gradle options wizard page problem1](https://user-images.githubusercontent.com/4611136/39491703-33a17b0e-4d5b-11e8-89c4-a5bcc9d00fdc.png)

**Comment by lukeu** [comment]
Huh, I encountered this just yesterday also. In my case I had not edited the preferences, instead I....

 * Imported a gradle-project and specified a local gradle installation.
 * Later, deleted the projects
 * Imported the same gradle-project

As per default, the step of the wizard shown above had the "Override" checkbox cleared - meaning "use the workspace settings". However, after clicking [Next >] it became apparent it was actually going to import it using the __local gradle installation__ again. There didn't seem to be a way to make it actually use the workspace settings again, I just had to override it with different settings. 

_(I hope I've remembered all this correctly.)_

**Comment by donat** [comment]
Yep, the problem is there, although it's already fixed on the 3.0 branch.
